castra/core.py

```diff
@@ -17,6 +17,18 @@
 from pandas import msgpack
 
 
+bp_args = bloscpack.BloscpackArgs(offsets=False, checksum='None')
+
+def blosc_args(dt):
```

Comment:
Not sure about these defaults. Should experiment a bit to find good ones. Shuffle can work well on floats too.